src/libexpr/eval.hh

```diff
@@ -615,7 +620,11 @@ public:
         return BindingsBuilder(*this, allocBindings(capacity));
     }
 
-    void mkList(Value & v, size_t length);
+    ListBuilder buildList(size_t size)
```

Comment:
Thought: builder pattern usually _ends_ with a call like `build`, so I found the call sites to look a bit weird.
It's consistent with `buildBindings` though.